<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1032</width>
     <height>900</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="s21::scene" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_line_width">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="prefix">
            <string>Line width: </string>
           </property>
           <property name="maximum">
            <double>2147483648.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_move_y">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="accelerated">
            <bool>false</bool>
           </property>
           <property name="prefix">
            <string>Move Y: </string>
           </property>
           <property name="minimum">
            <double>-2147483648.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2147483647.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_yRot">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="prefix">
            <string>Rotate Y : </string>
           </property>
           <property name="minimum">
            <double>-10000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="title">
              <string/>
             </property>
             <widget class="QRadioButton" name="radioButton_dotted">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>211</width>
                <height>23</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <underline>false</underline>
                <strikeout>false</strikeout>
                <stylestrategy>PreferDefault</stylestrategy>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Dotted</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_dash_dotted">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>10</y>
                <width>211</width>
                <height>23</height>
               </rect>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>dash-dotted</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_solid">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>10</y>
                <width>181</width>
                <height>23</height>
               </rect>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string> Solid</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_reset">
              <property name="geometry">
               <rect>
                <x>500</x>
                <y>10</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>RESET</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_ortho">
              <property name="geometry">
               <rect>
                <x>300</x>
                <y>10</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>Ortho</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_frustum">
              <property name="geometry">
               <rect>
                <x>400</x>
                <y>10</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>Frustum</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="title">
              <string/>
             </property>
             <widget class="QRadioButton" name="background">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>211</width>
                <height>23</height>
               </rect>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Background color</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QRadioButton" name="line">
              <property name="geometry">
               <rect>
                <x>220</x>
                <y>10</y>
                <width>211</width>
                <height>23</height>
               </rect>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Line color</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="point">
              <property name="geometry">
               <rect>
                <x>430</x>
                <y>10</y>
                <width>206</width>
                <height>23</height>
               </rect>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Point color</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_zRot">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="prefix">
            <string>Rotate Z: </string>
           </property>
           <property name="minimum">
            <double>-10000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_point_size">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="prefix">
            <string>Point Size: </string>
           </property>
           <property name="maximum">
            <double>2147483648.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_move_z">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="prefix">
            <string>Move Z: </string>
           </property>
           <property name="minimum">
            <double>-2147483648.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2147483647.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_move_x">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="prefix">
            <string>Move X: </string>
           </property>
           <property name="minimum">
            <double>-2147483648.000000000000000</double>
           </property>
           <property name="maximum">
            <double>2147483647.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_zoom">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="prefix">
            <string>Zoom: </string>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000001000000000</double>
           </property>
           <property name="maximum">
            <double>2147483647.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_xRot">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="prefix">
            <string>Rotate X: </string>
           </property>
           <property name="minimum">
            <double>-1000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_zoom">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_openFile">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Open File</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="4">
        <widget class="QLabel" name="label_info">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_screen">
           <property name="text">
            <string>Screen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_hide_show_all">
           <property name="text">
            <string>Hide/show all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_gif">
           <property name="text">
            <string>Gif</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>Point Type</string>
           </property>
           <widget class="QRadioButton" name="radioButton_empty">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>80</y>
              <width>91</width>
              <height>23</height>
             </rect>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>empty</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_square">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>91</width>
              <height>23</height>
             </rect>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>square</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_circle">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>50</y>
              <width>91</width>
              <height>23</height>
             </rect>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>circle</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5" rowspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="3" colspan="4">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="s21::ColorPalette" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::scene</class>
   <extends>QWidget</extends>
   <header>scene.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>s21::ColorPalette</class>
   <extends>QWidget</extends>
   <header>colorpalette.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
